start-chrome-for-playr.sh

```diff
@@ -35,7 +39,88 @@ no_nagging_options="--disable-features=SameSiteByDefaultCookies,CookiesWithoutSa
 # The path to the page that will check internet connection
 # before loading the actual signage channel
 # NOTE: check the location of the player_loader.html in the following line
-playr_loader_file="$HOME/playr-loader/playr_loader.html"
+execution_path=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+playr_loader_file="${execution_path}/playr_loader.html"
+
+# Add some terminal colors
+COLOR_OFF='\033[0m'       # Text reset
+COLOR_RED='\033[0;31m'    # Red
+COLOR_YELLOW='\033[0;33m' # Yellow
+COLOR_BLUE='\033[0;34m'   # Blue
+COLOR_GREEN='\033[0;32m'  # Green
+
+##########################################################################
+#							   METHODS     								 #
+##########################################################################
+
+# Use this to write error messages to the terminal
+log_error() {
+	echo -e "[ERROR] - $(date +%F-%T) - $COLOR_RED${1}$COLOR_OFF"
+}
+
+# parse mac address from the ip link command
+# ignore loopback devices and focus on link/ether
+# extract the mac and select the first one based on activation order in kernel
+parse_mac_from_ip_link() {
+	echo $(ip link | grep -E "link/ether" | grep -o -E '([[:xdigit:]]{2}:){5}[[:xdigit:]]{2}' | head -1)
+}
+
+# get first mac of the network hardware
+# order defined by kernel activation order
+get_first_hardware_mac() {
+	if ! which ip >/dev/null; then
+		echo "ip not installed on this system, please install ip"
+		exit 1
+	fi
+
+	parse_mac_from_ip_link
+}
+
+# get system uuid based on ioreg or ip link mac address
+get_system_uuid() {
+	local result=""
+
+	if [ "$(uname)" == "Darwin" ]; then
+		# get platform serial number, parse and strip quotes
+		result=$(ioreg -rd1 -c IOPlatformExpertDevice | awk '/IOPlatformSerialNumber/' | grep -o -E '("\w+")$' | sed -E 's/"//g')
+	else
+		result=$(get_first_hardware_mac)
+	fi
+
+	# check return code on error
+	if [ "$?" -ne "0" ]; then
+		echo "failed to retrieve system_uuid reason: $result"
+		exit 1
+	else
+		echo $result
+	fi
+}
+
+# return the path for the browser based on operating system
+get_chrome_browser() {
```

Comment:
Laten we dit get_browser maken en uitzoeken welke browser er geinstalleerd is, de locatie hoeft waarschijnlijk ook niet altijd die te zijn die hier gebruikt worden. 
Complicatie hierbij is dat de browser die we vinden ook invloed heeft op het voorkomen van meldingen bij het starten van de browser die later in het script wordt uitgevoerd